### ISSUE TITLE
Uploading an image removes all images from the page

### ISSUE THREAD

**Issue by jonshippling** [EJDinki/messe#92]
make that all of the website images on the server

**Comment by EJDinki** [comment]
Haven't looked at this yet. But if you just drop pictures in the right directory, do they show up on the site?

If so could a potential fix, if we don't get this working, be to just tell them to drop their pictures in a certain folder on the laptop (even shortcut the folder for them to the desktop) instead of uploading them?

**Comment by ajm5774** [comment]
I guess so. But i want to figure what this is happening and fix the root cause if we can. Lets see if its an easy fix first before hacking it

**Comment by EJDinki** [comment]
Looking into this now.

Online I found this:
http://forums.asp.net/t/1921010.aspx?What+is+the+MAX+file+upload+size+limit+in+ASP+net+4+0+

Saying the default max size is 4Mb.

I have not been able to replicate this problem. I can upload images fine. If I upload an image over 4Mb (I found a 10Mb one online) it directs me to an error page saying "max request length exceeded". If I hit the back button though I'm brought back to multimedia and everything seems intact. We can change it so it allows more than 4Mb, though that seems pretty reasonable given that the pictures they gave us are around 2Mb and they are only ever shown as small pictures on the site. I'd be happy with keeping the default 4Mb.

The only actions taken by the upload method are file.SaveAs. I don't see how anything destructive could happen, especially deleteing all pictures.

Could someone see if they can reproduce this? I'm not sure what I can do.
